pandas/core/generic.py

```diff
@@ -7715,34 +7715,107 @@ def last(self, offset):
     def rank(self, axis=0, method='average', numeric_only=None,
              na_option='keep', ascending=True, pct=False):
         """
-        Compute numerical data ranks (1 through n) along axis. Equal values are
-        assigned a rank that is the average of the ranks of those values
+        Compute numerical data ranks (1 through n) along axis.
+
+        By default, equal values are assigned a rank that is the average of the
+        ranks of those values.
 
         Parameters
         ----------
         axis : {0 or 'index', 1 or 'columns'}, default 0
-            index to direct ranking
-        method : {'average', 'min', 'max', 'first', 'dense'}
-            * average: average rank of group
-            * min: lowest rank in group
-            * max: highest rank in group
-            * first: ranks assigned in order they appear in the array
-            * dense: like 'min', but rank always increases by 1 between groups
-        numeric_only : boolean, default None
+            Index to direct ranking.
+        method : {'average', 'min', 'max', 'first', 'dense'}, default 'average'
+            Which method to use to rank equal values:
```

Comment:
Hi @WillAyd thank you very much for the comments.

I have added this because of the docstring validator failing as well.
I will remove it

Reply:
Hmm OK I didn't even think about that aspect of it. With that said, I'd say go ahead and leave in place. Copying @datapythonista in case this isn't desired behavior for the validator

Reply:
We have pending in the validator to add support for both, lists at the beginning or at the end of the parameter description. So ignore the validator error with the version that makes more sense to you.

Reply:
Thank you both for the help!